src/pages/Share/SubmitTab.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import {saveUnknownUserDetails} from '@libs/actions/Share';
+import Navigation from '@libs/Navigation/Navigation';
+import MoneyRequestParticipantsSelector from '@pages/iou/request/MoneyRequestParticipantsSelector';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+
+function SubmitTab() {
+    return (
+        <MoneyRequestParticipantsSelector
+            iouType={CONST.IOU.TYPE.SUBMIT}
+            onFinish={() => {}}
+            onParticipantsAdded={(value) => {
+                const participant = value.at(0);
+                const reportID = participant?.reportID;
+                const accountID = participant?.accountID;
+                if (accountID && !reportID && participant) {
+                    saveUnknownUserDetails(participant);
+                }
+                Navigation.navigate(ROUTES.SHARE_SUBMIT_DETAILS.getRoute(`${!reportID ? accountID : reportID}`));
```

Comment:
This looks odd to me, why would `accountID` be passed to screen that is supposed to have `reportID` in the url? `'share/share-details/:reportID'`

Reply:
I propose renaming it to something like `reportOrAccountID` (or just `ID`). If `reportID` is always present, and this fallback is added to silence TS, I would rather throw an error or do nothing (simple early return)

Reply:
Also, I recommend using nullish coalescing:
```ts
Navigation.navigate(ROUTES.SHARE_SUBMIT_DETAILS.getRoute(reportID ?? accountID));
```